src/content/contributing/style-guide/index.md

```diff
@@ -146,6 +146,19 @@ Proof-of-work should be capitalized at the beginning of a sentence. In any other
 
 The same rules we apply to proof-of-work are applicable to proof-of-stake, proof-of-authority, proof-of-humanity, proof-of-individuality, etc.
 
+### Smart Contract {#smart-contract}
+
+Smart contract should be capitalized at the beginning of a sentence. In any other instance, all letters should be lower case.
```

Comment:
```suggestion
Smart contract is a common noun and should only be capitalized at the beginning of a sentence. In any other instance, all letters should be lowercase.
```

Reply:
'lower case' should be 'lowercase' on the 'Zero-knowledge' section also (as should be the 'is a common noun' part). Should I add those too, do I need to open another PR to do that, or can I do it on this one somehow?